cmd/soroban-cli/src/commands/contract/info.rs

```diff
@@ -49,6 +50,9 @@ pub enum Cmd {
     ///
     /// Outputs no data when no data is present in the contract.
     EnvMeta(env_meta::Cmd),
+
+    /// Get the wasm hash of a deployed contract
+    WasmHash(wasm_hash::Cmd),
```

Comment:
@fnando wdyt naming this command `executable`? Technically contracts can have an executable that is either:
- stellar-asset
- wasm-hash

In the past when we've made commands that accepted a Contract ID and errored on stellar-assets, it's been annoying that the edge case wasn't supported.

The command could output not just the hash but the type of executable, maybe in a simple json format. Or just use xdr-json of the ContractExecutable xdr type.

Reply:
It also might be fair that getting a wasm-hash explicitly as the only output could be more helpful when scripting in some cases. There might be a case for both commands co-existing.